ssc/Pos/Ssc/Toss/Logic.hs

```diff
@@ -99,7 +102,7 @@ rollbackSsc oldestEOS (NewestFirst payloads)
         resetShares
     | otherwise = do
         setEpochOrSlot (pred oldestEOS)
-        mapM_ rollbackSscDo payloads
+        mapM_ (rollbackSscDo . bcmPayload) payloads
```

Comment:
`bcmPayload` is a partial function, it will fail here if payload is `ComponentBlockGenesis`.